matchms/importing/load_from_msp.py

```diff
@@ -140,11 +146,14 @@ def parse_metadata(rline: str, params: dict):
             if splitted_line[0].lower().strip() in params.keys() and splitted_line[0].lower().strip() == 'smiles':
                 params[splitted_line[0].lower()+"_2"] = splitted_line[1].strip()
             else:
-                params[splitted_line[0].lower().strip()] = splitted_line[1].strip()
+                params[splitted_line[0].lower().strip()
+                       ] = splitted_line[1].strip()
     else:
         params[splitted_line[0].lower()] = splitted_line[1].strip()
 
 
 def contains_metadata(rline: str) -> bool:
     """ Check if line contains Spectrum metadata."""
-    return ':' in rline
+    has_colon = ':' in rline
+    is_golm_peaks = re.match(r"(\d+:{1}\d+)", rline) is not None
```

Comment:
Could you add a short description above this line? If I understand correctly the golm format can have ":" in a line which is a peak and not metadata right?

Reply:
Yeah exactly. GOLM specifies parks as mz:int, so for example 55:1290. It conflicts with the previous recognition of a metadata line. The regex though should only find number:number sections in strings, which are more than unlikely to occur in comments I think.

Reply:
Adressed in https://github.com/matchms/matchms/pull/393/commits/7ef2c57896b08fa74c8128d038f65eed9657c820